modules/installation/proc-installing-rhdh-on-kubernetes-with-helm-cli.adoc

```diff
@@ -0,0 +1,75 @@
+[id="installing-rhdh-on-kubernetes-with-helm-cli_{context}"]
+= Installing {product} on Kubernetes with Helm CLI
+
+You can use the Helm CLI to install {product} on the following platforms:
+
+* {eks-brand-name} ({eks-short})
+* {aks-brand-name} ({aks-short})
+
+.Prerequisites
+* A `kubectl` session, with `developer` or `admin` permissions, on a Kubernetes cluster.
+* You have installed the Helm CLI.
+
+.Procedure
+. Add the OpenShift Helm Charts repository.
++
+----
+helm repo add openshift-helm-charts https://charts.openshift.io/
+----
+
+. Define the _<rhdh>_ Kubernetes namespace name to deploy to:
```

Comment:
I know the style guide says "make every single step its own code snippet" :jackie-chan-why: but this is super verbose/dumb. 

I would strongly recommend making this a single 3 line block. Or it could be a single line.